contentcuration/contentcuration/frontend/channelEdit/views/files/FileUploadItem.vue

```diff
@@ -167,11 +192,20 @@
       uploadingHandler(fileUpload) {
         this.fileUploadId = fileUpload.id;
       },
+      downloadFile() {
+        window.open(this.fileDisplay.url, '_blank');
```

Comment:
Could we have the download option initiate a download? For PDFs this will likely just open it in a tab, meaning actually downloading it isn't a simple click of this option. Here's an example of an anchor, using `download` and the original file name, if applicable.

![image (38)](https://github.com/user-attachments/assets/82c6af82-9537-485e-99b5-6a1ddf5913ba)